airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -58,21 +67,60 @@ public AbstractJdbcDestination(final String driverClass,
     this.sqlOperations = sqlOperations;
   }
 
-  @Override
-  public AirbyteConnectionStatus check(final JsonNode config) {
-    final DataSource dataSource = getDataSource(config);
+  /**
+   * Returns any schemas which may be written to and need verification
+   * @param config configuration for the destination which contains default schemas
+   * @param catalog {@link ConfiguredAirbyteCatalog} which may contain options for overriding defaults
+   * @return a set of schema names to run through verification
+   */
+  protected Set<String> getOutputSchemas(final JsonNode config, final ConfiguredAirbyteCatalog catalog) {
+    if (sqlOperations.isSchemaRequired()) {
+      Preconditions.checkState(config.has(JdbcUtils.SCHEMA_KEY), "jdbc destinations must specify a schema.");
+    }
+    final String schemaDefaultConfigKey = sqlOperations.isSchemaRequired() ? JdbcUtils.SCHEMA_KEY : JdbcUtils.DATABASE_KEY;
+    final String defaultSchemaName = namingResolver.getIdentifier(config.get(schemaDefaultConfigKey).asText());
+    final Set<String> outputSchemas = new HashSet<>();
+    outputSchemas.add(defaultSchemaName);
```

Comment:
nitpick: I think (?) we only need to check the default schema if any stream has null namespace. Not actually 100% what the configured catalog looks like in this case.